tests/fast-integration/installer/helm.js

```diff
@@ -45,6 +45,9 @@ async function helmInstallUpgrade(release, chart, namespace, values, profile) {
     release,
     chart,
   ];
+  if (additionalArgs) {
+    args = args.concat(additionalArgs);
```

Comment:
You can also write this following way:
```
args.push(...additionalArgs)
```
Following this approach args can still be declared as `const`.